packages/editor/src/client/blocks/embeds/iframe.tsx

```diff
@@ -6,23 +6,43 @@ export interface IframeEmbedProps {
   title?: string
   width?: number
   height?: number
+  styleHeight?: string
+  styleWidth?: string
+  ratio?: boolean
 }
 
-export function IframeEmbed({url, title, width = 300, height = 300}: IframeEmbedProps) {
-  const ratio = width / height
-
+export function IframeEmbed({
+  url,
+  title,
+  width = 300,
+  height = 300,
+  styleHeight,
+  styleWidth,
+  ratio
+}: IframeEmbedProps) {
+  const ratioVal = width / height
+  const styles = {}
+  if (!ratio) {
+    Object.assign(styles, {
+      width: styleWidth,
+      height: styleHeight
+    })
+  }
   return (
     <Box width="100%">
-      <Box position="relative" paddingTop={`${(1 / ratio) * 100}%`} width="100%">
+      <Box
+        position="relative"
+        paddingTop={`${ratio || styleHeight === undefined ? (1 / ratioVal) * 100 + '%' : '0'}`}
```

Comment:
I don't understand this logic

Reply:
by default `useRatio` or `ratio` is true which means the padding is calculated BUT calc of padding is not always fitting and results to problems

Reply:
replaced by more explainable vars and logic